samples/pipeline_template_examples/02_basic_cpu_pipeline_tomo_standard.yaml

```diff
@@ -36,7 +36,7 @@
     find_center_vo:
       data_in: tomo
       data_out: cor
-      ind: null
```

Comment:
@yousefmoazzam do you know why **ind** is generated as _null_ instead of _None_ as in [TomoPy function](https://tomopy.readthedocs.io/en/stable/api/tomopy.recon.rotation.html#tomopy.recon.rotation.find_center_vo)?